src/app/docs/tabs/tabs-docs.component.html

```diff
@@ -4,10 +4,373 @@
+      </sky-docs-design-guideline>
+
+    </sky-docs-demo-page-section>
+      
+    <sky-docs-demo-page-section
+      heading="Behavior & states"
+    >
+    
+        <sky-docs-design-guideline
+        heading="Overflowing tabs"
+      >
+        <p>
+          If the tab set includes too many tabs to fit in the tab bar's available horizontal space, the tabs collapse into a dropdown.
+        </p>
+        <sky-docs-design-guideline-thumbnail
+          imageSource="~/assets/img/guidelines/tabs/tabs-behavior-1.png"
+        >
+        </sky-docs-design-guideline-thumbnail>    
+        </sky-docs-design-guideline>
+        
+        <sky-docs-design-guideline
+        heading="States"
+      >
+        <table class="sky-docs-table">
+          <tr>
+            <td class="sky-docs-table-cell sky-docs-table-cell-primary">
+              Selected
+            </td>
+            <td class="sky-docs-table-cell">
+              <sky-docs-design-guideline-thumbnail
+                imageSource="~/assets/img/guidelines/tabs/tabs-state-selected.png"
+              >
+              </sky-docs-design-guideline-thumbnail>
+            </td>
+          </tr>
+          <tr>
+            <td class="sky-docs-table-cell">
+              Unselected
+            </td>
+            <td class="sky-docs-table-cell">
+              <sky-docs-design-guideline-thumbnail
+                imageSource="~/assets/img/guidelines/tabs/tabs-state-unselected.png"
+              >
+              </sky-docs-design-guideline-thumbnail>
+            </td>
+          </tr>
+          <tr>
+            <td class="sky-docs-table-cell">
+              Hover
+            </td>
+            <td class="sky-docs-table-cell">
+              <sky-docs-design-guideline-thumbnail
+                imageSource="~/assets/img/guidelines/tabs/tabs-state-hover.png"
+              >
+              </sky-docs-design-guideline-thumbnail>
+            </td>
+          </tr>
+          <tr>
+            <td class="sky-docs-table-cell">
+              Collapsed, closed
+            </td>
+            <td class="sky-docs-table-cell">
+              <sky-docs-design-guideline-thumbnail
+                imageSource="~/assets/img/guidelines/tabs/tabs-state-collapsed-closed.png"
+              >
+              </sky-docs-design-guideline-thumbnail>
+            </td>
+          </tr>
+          <tr>
+            <td class="sky-docs-table-cell">
+              Collapsed, open
+            </td>
+            <td class="sky-docs-table-cell">
+              <sky-docs-design-guideline-thumbnail
+                imageSource="~/assets/img/guidelines/tabs/tabs-state-collapsed-open.png"
+              >
+              </sky-docs-design-guideline-thumbnail>
+            </td>
+          </tr>
+        </table>
+      </sky-docs-design-guideline>
+        
+    </sky-docs-demo-page-section>
+      
+    <sky-docs-demo-page-section
+      heading="Content"
+    >
+    
+        <div class="sky-text-block">
```

Comment:
@blackbaud-johnly Can probably tweak/reuse the updated language from the Page Design guidelines re: tabbed page titles here.

Reply:
Also, I'll be migrating the `.sky-text-block` CSS class to docs-tools as `.sky-docs-text-block`. Hopefully will have a release out this week.

Reply:
Anyone know what I'm doing wrong here? For some reason the `<sky-code>` tag seems to break.
![image](https://user-images.githubusercontent.com/15896944/73080472-e1585a80-3e93-11ea-9637-e4ddebff3d8a.png)

Reply:
What do you mean by "break"? Did you add the `SkyCodeModule` to the app-extras.module.ts `exports` array?

Reply:
(Also, @Blackbaud-AlexKingman I've added the `.sky-docs-text-block` class to version `3.0.0` of docs-tools.)

Reply:
> What do you mean by "break"? Did you add the `SkyCodeModule` to the app-extras.module.ts `exports` array?

I did not.

Reply:
@Blackbaud-SteveBrush, it looks like we still need to add `SkyCodeModule` to the exports array. ... Do I just add it to the list at https://github.com/blackbaud/skyux-tabs/blob/master/src/app/app-extras.module.ts#L33? Or is there anything else that needs to be done?

Reply:
You're correct, @blackbaud-johnly.

Reply:
Great. Thanks. I'll make the change...